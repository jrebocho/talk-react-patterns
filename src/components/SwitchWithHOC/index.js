import withBlueClick from '../BlueClickHOC'
import StatelessSwitch from '../StatelessSwitch'

export default withBlueClick(StatelessSwitch)
