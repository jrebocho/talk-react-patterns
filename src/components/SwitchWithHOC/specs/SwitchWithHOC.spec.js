import React from 'react'
import { shallow } from 'enzyme'
import SwitchWithHOC from '../../SwitchWithHOC'

describe('<SwitchWithHOC />', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<SwitchWithHOC />)
    expect(wrapper).toMatchSnapshot()
  })
})
