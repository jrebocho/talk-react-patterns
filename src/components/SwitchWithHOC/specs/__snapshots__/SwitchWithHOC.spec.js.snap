// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SwitchWithHOC /> renders correctly 1`] = `
<div
  className="blue-box"
  onClick={[Function]}
>
  <StatelessSwitch
    on={false}
  />
</div>
`;
